make/modules/jdk.accessibility/Lib.gmk

```diff
@@ -31,84 +31,50 @@ ifeq ($(call isTargetOs, windows), true)
   ACCESSIBILITY_SRCDIR := $(TOPDIR)/src/jdk.accessibility/windows/native
 
   ##############################################################################
-  # Setup rules to create 32/64 bit version of javaaccessbridge
-  #
-  # Parameter 1 Suffix
-  # Parameter 2 ACCESSBRIDGE_ARCH_ suffix
+  ## Build libjavaaccessbridge
   ##############################################################################
-  define SetupJavaDLL
-    $(call SetupJdkLibrary, BUILD_LIBJAVAACCESSBRIDGE$1, \
-        NAME := javaaccessbridge$1, \
-        SRC := libjavaaccessbridge, \
-        EXTRA_SRC := common, \
-        OPTIMIZATION := LOW, \
-        DISABLED_WARNINGS_microsoft := 4311 4302 4312, \
-        CXXFLAGS_FILTER_OUT := -MD, \
-        CXXFLAGS := -MT -DACCESSBRIDGE_ARCH_$2, \
-        EXTRA_HEADER_DIRS := \
-            include/bridge \
-            java.desktop:include, \
-        JDK_LIBS := java.desktop:libjawt, \
-        LIBS_windows := advapi32.lib comdlg32.lib gdi32.lib kernel32.lib \
-            odbc32.lib odbccp32.lib ole32.lib oleaut32.lib shell32.lib \
-            user32.lib uuid.lib winspool.lib, \
-        VERSIONINFO_RESOURCE := \
-            $(ACCESSIBILITY_SRCDIR)/common/AccessBridgeStatusWindow.rc, \
-    )
 
-    TARGETS += $$(BUILD_LIBJAVAACCESSBRIDGE$1)
-  endef
+  $(eval $(call SetupJdkLibrary, BUILD_LIBJAVAACCESSBRIDGE, \
+      NAME := javaaccessbridge, \
+      EXTRA_SRC := common, \
+      OPTIMIZATION := LOW, \
+      DISABLED_WARNINGS_microsoft := 4311 4302 4312, \
+      CXXFLAGS_FILTER_OUT := -MD, \
+      CXXFLAGS := -MT -DACCESSBRIDGE_ARCH_64, \
+      EXTRA_HEADER_DIRS := \
+          include/bridge \
+          java.desktop:include, \
+      JDK_LIBS := java.desktop:libjawt, \
+      LIBS_windows := advapi32.lib comdlg32.lib gdi32.lib kernel32.lib \
+          odbc32.lib odbccp32.lib ole32.lib oleaut32.lib shell32.lib \
+          user32.lib uuid.lib winspool.lib, \
+      VERSIONINFO_RESOURCE := \
+          $(ACCESSIBILITY_SRCDIR)/common/AccessBridgeStatusWindow.rc, \
+  ))
+
+  TARGETS += $(BUILD_LIBJAVAACCESSBRIDGE)
 
   ##############################################################################
```

Comment:
Most of the desktop related changes are related to Assistive Technologies
I don't think we currently provide a 32-bit windowsaccessbridge.dll in the 64 bit JDK, but I'd like to be sure I am not forgetting something.
The point being windowsaccessbridge.dll is not loaded by the JDK, but by an AT, so traditionally we provided both 32 and 64 bit versions because we don't control that AT.

So I would like Alex Zuev to review these changes. For whatever reason his git hub handle doesn't seem to be found. I think it is something like @azuev-java

Reply:
We built 32-bit dll in order to provide access to the accessibility interfaces for the legacy 32-bit software that can not load the 32-bit code. We abandoned this practice since at least Java 11 and we had no complaints about it ever since. All the relevant accessibility software we are aware of have 64-bit executable and only support 32-bit operating systems with the legacy versions that are not recommended to use with modern OSes. I do not see any problem in abandoning 32-bit code in windowsaccessbridge.dll.

Reply:
@azuev-java Thanks! I have one more question for you: To avoid risking breaking any compatibility, the file generated from the source code in `windowsaccessbridge` is still compiled into a file called `windowsaccessbridge-64.dll`.

This is a bit unusual, and requires a quirk in the build system -- normally we assume there is a 1-to-1 relationship between the directory containing the native library source code, and the generated `.dll` file.

Is this file exposed to external parties, that rely on a specific name? Or is it just used internally by the JDK, so we could rename it to `windowsaccessbridge.dll`, and just update our reference to it?